app.go

```diff
@@ -38,16 +42,17 @@ func New(appID string, config *Config) (*App, error) {
 		client: client,
 	}
 
-	app.JWKS, err = app.fetchJWKS()
-	if err != nil {
+	// cached set setup taken from https://github.com/lestrrat-go/jwx/blob/8d1d78351e9f259723f9f558889a78e327379683/examples/jwk_cache_example_test.go#L21-L36
```

Comment:
This link can perhaps be omitted, as the example is surfaced in the package's docs [here](https://github.com/lestrrat-go/jwx/blob/develop/v3/docs/04-jwk.md#auto-refreshing-remote-keys).

But I'm also good with keeping it if it feels like a good point of reference.

Reply:
removed in [058e8d4](https://github.com/passageidentity/passage-go/pull/88/commits/058e8d4b47909b5cfc17d2f04faf7b43351443c0)